Tests/Tests/CDTSessionCookieInterceptorTests.m

```diff
@@ -49,18 +49,23 @@ - (void)setUp
       return [[request.URL host] isEqualToString:@"username.cloudant.com"];
     }
         withStubResponse:^OHHTTPStubsResponse *__nonnull(NSURLRequest *__nonnull request) {
-          return [OHHTTPStubsResponse
-              responseWithJSONObject:@{
-                  @"ok" : @(YES),
-                  @"name" : @"username",
-                  @"roles" : @[ @"_admin" ]
-              }
-                          statusCode:200
-                             headers:@{
-                                 @"Set-Cookie" :
-                                     [NSString stringWithFormat:@"%@; Version=1; Path=/; HttpOnly",
-                                                                testCookieHeaderValue]
-                             }];
+
+          if ([request.HTTPMethod isEqualToString:@"POST"]) {
+              return [OHHTTPStubsResponse
+                  responseWithJSONObject:@{
+                      @"ok" : @(YES),
+                      @"name" : @"username",
+                      @"roles" : @[ @"_admin" ]
+                  }
+                              statusCode:200
+                                 headers:@{
+                                     @"Set-Cookie" : [NSString
+                                         stringWithFormat:@"%@; Version=1; Path=/; HttpOnly",
+                                                          testCookieHeaderValue]
+                                 }];
+          } else {
+              return [OHHTTPStubsResponse responseWithJSONObject:@{} statusCode:200 headers:@{}];
```

Comment:
This is in the `else` block so it's not a POST to session. It looks like it'll return some info about the user whose details are in the request with a `200` code from the docs.